README.md

```diff
@@ -102,6 +113,13 @@ Once the backup is completed you should see the backup marked as `Completed`.
 
 
 ### Creating a restore from backup
+#### Creating a restore from local backup/snapshot
+To restore local snapshot, run the following command:
+
+`velero restore create --from-backup backup_name --restore-volumes=true --namespace-mappings source_ns:destination_ns`
```

Comment:
does namespace-mappings should work for restore of remote backup as well right?
or is this for next PR?

Reply:
no. It is not supported. I've added limitation. It requires handing of incremental backups.